src/Geta.NotFoundHandler/Infrastructure/Initialization/Upgrader.cs

```diff
@@ -74,102 +79,130 @@ private bool CreateRedirectsTable()
                                         [RedirectType] [int] NOT NULL,
                                         CONSTRAINT [PK_NotFoundHandlerRedirects] PRIMARY KEY CLUSTERED ([Id] ASC) ON [PRIMARY]
                                         ) ON [PRIMARY]";
-            var created = _dataExecutor.ExecuteNonQuery(createTableScript);
-            _logger.LogInformation("Create NotFoundHandler redirects table END");
+        var created = _dataExecutor.ExecuteNonQuery(createTableScript);
```

Comment:
if this will fail - an exception will be thrown or no?

Reply:
I am not sure, but I am following a pattern used for years in the project.

P.S. The file was reformatted, so all lines are now changed - these are not new changes.